saleor/plugins/webhook/tests/test_webhook.py

```diff
@@ -120,14 +120,12 @@ def test_trigger_webhooks_for_event_with_secret_key(
     expected_data = serialize("json", [order_with_lines])
     trigger_webhooks_for_event(WebhookEventType.ORDER_CREATED, expected_data)
 
-    expected_signature = create_hmac_signature(
-        expected_data, webhook.secret_key, "utf-8"
-    )
+    expected_signature = signature_for_payload(expected_data, webhook.secret_key)
     expected_headers = {
         "Content-Type": "application/json",
         "X-Saleor-Event": "order_created",
         "X-Saleor-Domain": "mirumee.com",
-        "X-Saleor-HMAC-SHA256": f"sha1={expected_signature}",
+        "X-Saleor-HMAC-SHA256": expected_signature,
```

Comment:
You are removing the `sha1=` prefix which is used in apps to identify the signature. If we don't need the prefix here we need to modify the signature handler in our services.

Reply:
I know, I've also opened a ticket to track this as this is actually a SHA2 signature, not SHA1.